fem/quadinterpolator.hpp

```diff
@@ -135,6 +135,10 @@ class QuadratureInterpolator
    /// Interpolate the values of the E-vector @a e_vec at quadrature points.
    void Values(const Vector &e_vec, Vector &q_val) const;
 
+   /// @brief Interpolate the physical values of the E-vector @a e_vec at
+   /// quadrature points.
+   void PhysicalValues(const Vector &e_vec, Vector &q_val) const;
```

Comment:
For consistency with `PhysDerivatives()`, it would be better to use the name `PhysValues()` maybe 🤔 .

Reply:
For consistency I think you're right (though I personally prefer the unabbreviated version).

Changed in a5e62babfdab6301b4b48f4c24b63f0fd1f0fd75.